README.md

```diff
@@ -32,28 +32,45 @@ Some overview description will go here
 
 > Or take a look at our detailed [getting started][getting-started] instructions.
```

Comment:
This artifact is still here pointing at the old set up instructions. Not sure if want to get rid of it or keep it there in case someone can't get vagrant on their system for whatever reason.

Reply:
The duplicate getting started guide was removed.